projects/orquestra-sdk/tests/integration/api/test_api_with_ray.py

```diff
@@ -5,7 +5,7 @@
 from unittest.mock import Mock
 
 import pytest
-from orquestra.workflow_runtime._testing import _connections
+from orquestra.workflow_runtime._testing import _connections  # type: ignore
```

Comment:
What's the reason for this typing ignore? Is it importing a private symbol?

WDYT about actually making `orquestra.workflow_runtime.testing` public? To me this seems like a similar situation we had with _shared_. We made it public because this guards us against depending on private modules, which makes it easier to figure out when we introduce backwards-incompatible changes.

Reply:
this # type: ignore is not because we have any issues with private/public modules.
Its comming from the fact that style-check we use do not have orquestra.workflow_runtime installed right now, so it marks all usages of it as invalid.

This can be fixed in 2 ways:
Easy way - is what I did
Hard way - make style-fix install orquestra.workflow_runtime - which is hard as we rely on the fact that it uses the same makefile recipes as test-coverage, which we intentionally do NOT want to have orquestra.workflow_runtime installed

Reply:
Ill create a ticket to transition to the "hard way". Now when im thinking about it, its better and more reliable.

Reply:
Nvm, agreeed offline how to proceed with this PR